src/Bass/Shared/ChannelReferences.cs

```diff
@@ -22,7 +22,7 @@ public static void Add(int Handle, int SpecificHandle, object proc)
         {
 #if !__IOS__
             // in .NET iOS, the __IOS__ constant cannot be seen, so rely on RuntimeFeature instead.
-            if (!RuntimeFeature.IsDynamicCodeCompiled)
+            if (!RuntimeFeature.IsDynamicCodeSupported)
```

Comment:
why is this changed?

Reply:
It's @frenzibyte's [PR which got merged into ManagedBass](https://github.com/ManagedBass/ManagedBass/pull/118), I think it would be better to have him weigh in here. But I'd like to clarify that its in ManagedBass master.

This current PR both adds support for Volume FX and BassLoud, and also gets us up to date with ManagedBass master.

Reply:
It seems this specific change was not included in our fork. It was discussed in https://github.com/ManagedBass/ManagedBass/pull/118#discussion_r1254046836 if you want to have a read there. 

TL;DR this change only affects osu! on release configuration with phone deployment and osu!/osu-framework on debug configuration with both phone deployment and via simulator.

I would recommend discarding this change from this PR, it's not relevant and I don't think anyone is willing to test and confirm whether it's better in practice.